### PR TITLE
Add cmdlet Test-AzMySqlFlexibleServerConnect

### PR DESCRIPTION
**Pull request by msJinLei**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1 [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy.
+        ${ProxyUseDefaultCredentials}
+    )
+
+    process {
+        if (!(Get-Module -ListAvailable -Name SimplySQL)){
+            Write-Error "This cmdlet requires SimplySQL module. Please install the module first by running Install-Module -Name SimplySQL."
```
Comment (wyunchi-ms) [inline src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1:109]: Neet to exit here.

File: src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1 [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        }
+        Import-Module SimplySQL
+        
+        $AdministratorUserName = [string]::Empty
+        if ($PSBoundParameters.ContainsKey('AdministratorUserName')) {
+            $AdministratorUserName = $PSBoundParameters.AdministratorUserName
+            $null = $PSBoundParameters.Remove('AdministratorUserName')
+        }
```
Comment (wyunchi-ms) [inline src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1:117]: I suggest to move this block to line 135

File: src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1 (hunk 1 above)
Comment (msJinLei) [reply src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1:109]: @DaeunYim could you modify it in the generation branch?

File: src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1 (hunk 2 above)
Comment (msJinLei) [reply src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1:117]: @DaeunYim could you modify it in the generation branch?

File: src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1 (hunk 1 above)
Comment (DaeunYim) [reply src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1:109]: Made the change for both mysql and pg

File: src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1 (hunk 2 above)
Comment (DaeunYim) [reply src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1:117]: Made the change for both mysql and pg

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**